src/store/home/home.reducer.js

```diff
@@ -111,15 +112,17 @@ function homeReducer (state = initialHomeState, action) {
           data: {
             ...state.accountsTask.data,
             accounts: action.data.accounts,
-            pagination
+            pagination,
+            fromItemHistory
           }
         }
       }
     }
     case homeActionTypes.LOAD_POOL_TRANSACTIONS: {
       return {
         ...state,
-        poolTransactionsTask: state.poolTransactionsTask.status === 'pending'
+        poolTransactionsTask: state.poolTransactionsTask.status === 'pending' ||
+          state.poolTransactionsTask.status === 'failed'
```

Comment:
This is not correct, but I've already fixed it in the typescript migration branch.